### ISSUE TITLE
Fetal intracranial hemorrhage

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7204]
**Preferred term label:**
Fetal intracranial hemorrhage

**Synonyms**

**Definition (free text, please give PubMed ID)**

Fetal intracranial hemorrhage
-- Fetal intraventricular hemorrhage
-- Fetal cerebral parenchymal hemorrhage
-- Fetal pericerebral hemorrhage (in the extra-axial space)
-- Fetal cerebellar, vermian, cerebellar hemisphere hemorrhage (for U/S focal hyperechoic lesion consistent with hemorrhage)

Vascular malformation-associated hemorrhagic lesion

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by patt1morgan** [comment]
@pnrobinson 
Hi Peter,  This is the best information I could find....but, should Intracerebral and extracerebral be considered 'children" of ICH, and have their own HPO id?

Intracranial hemorrhage (ICH) is rare in the fetus.  Intracranial hemorrhages (ICHs) are subdivided into intracerebral and extracerebral (subdural hematomas). Intracerebral hemorrhages are further subdivided into intraventricular and infratentorial (occurring within the posterior fossa). Factors associated with in utero ICH include a variety of conditions, mostly maternal trauma and fetal coagulation disorders. In many cases, however, the cause is not identified. PMID: 27734711  PMID: 12905503 

Intraventricular hemorrhages are categorized following the classification commonly used in neonates as follows:
Grade I: limited to subependymal matrix;

Grade II: clear spill-over to ventricles, but filling less than 50% of the lateral ventricle and without ventriculomegaly;

Grade III: spill-over to the ventricle, with flooding of 50% or more of one or both lateral ventricles and ventriculomegaly; and

Grade IV: Grades I, II or III with hemorrhage in a large part of the periventricular parenchyma  PMID: 12905503 (includes images)

Synonyms: fetal stroke, prenatal cerebrovascular disorder perinatal stroke and perinatal brain injury, fetal intracranial bleed  PMID: 27734711

**Comment by pnrobinson** [comment]
The above classification (i-IV) is used for postnatal findings and is not what we want for fetal findings.

This is good--
Fetal intracranial hemorrhage
-- Fetal intraventricular hemorrhage
-- Fetal cerebral parenchymal hemorrhage
-- Fetal pericerebral hemorrhage (in the extra-axial space)
-- Fetal posterior fossa hemorrhage
-------vermian
-------cerebellar hemisphere hemorrhage 

To be discussed further @ferdho

**Comment by pnrobinson** [comment]
THe above is consensus

**Comment by jagcsun** [comment]
Just came across a cerebral palsy patient clinical summary with antenatal IVH; since seen this term already listed for new term request in HPO, sharing here some relevant notes& references:

Fetal intracranial hemorrhage (Fetal ICH) is a pathophysiological process involving haemorrhagic and hypoxic–ischaemic insults resulting in antenatal brain damage (PMID: 34765420).

Vascular connection between germinal matrix and the subependymal venous network, which are bleeding sites in the Fetal intraventricular hemorrhage (Fetal IVH), is clearly present only after 20 weeks of gestation (PMID: 33920939).

**Comment by pnrobinson** [comment]
Adding terms with the following definitions
Fetal intracranial hemorrhage 
HP:0034208
def: Abnormal bleeding inside the skull (cranium) of a fetus.
Comment: Fetal intracranial hemorrhage occurs when a blood vessel within or adjacent to the brain ruptures or leaks, resulting in bleeding into the brain or surrounding area. Fetal intracranial hemorrhage can result in disruption of blood flow to the developing brain (ischemia). Addditionally, accumulation of blood can compress brain tissue.

**Comment by pnrobinson** [comment]
Fetal subependymal hemorrhage
HP:0034209
synonym:Grade I fetal intracranial hemorrhage
def:A type of fetal intracranial hemorrhage located in the subependymal germinal matrix below the frontal horns of the lateral ventricles in the caudothalamic notch.
comment:Subependymal hemorrhage originates from the germinal matrix and appears on sonography as echogenic material located below the frontal horns in the floor of the lateral ventricle within the caudothalamic notch. Acute hemorrhage appears sonographically as echogenic and homogeneous; the hemorrhage changes over time to a more complex appearance with an echogenic external lining and internal sonolucent center and then eventually becomes cystic in nature.

**Comment by pnrobinson** [comment]
Fetal extra-axial hemorrhage
def: A type of fetal intracranial hemorrhage with bleeding that occurs within the skull but outside of the brain tissue itself (into the epidural, subdural, or arachnoid space).

**Comment by pnrobinson** [comment]
Fetal posterior fossa hemorrhage
HP:0034213
Def: A type of fetal intracranial hemorrhage that occurs in the posterior cranial fossa, which is the part of the cranial cavity that is located between the foramen magnum and tentorium cerebelli and contains the brainstem and cerebellum.

**Comment by pnrobinson** [comment]
Grade II fetal intraventricular hemorrhage
HP:0034214
def: A type of Fetal intraventricular hemorrhage characterized by clear hemorrhage spillover to the ventricles filling less than 50% of the lateral ventricle and without ventriculomegaly of greater than 15 mm in the transverse diameter of the lateral ventricular atrium.
PMID:16082722

**Comment by pnrobinson** [comment]
Grade III fetal intraventricular hemorrhage
def: A type of fetal intraventricular hemorrhage (IVH) characterized by spillover to ventricles with flooding of 50% or more of one or both lateral ventricles accrued accompanying ventriculomegaly, but with no apparent brain parenchymal injury. This type of IVH may be defined as a diameter exceeding 15 mm in the transverse diameter of the lateral ventricular atrium as visualiuzed by prenatal sonography.
PMID:16082722

**Comment by pnrobinson** [comment]
these terms were added, closing.
